api-authorization/server/lib/authorization-middleware.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import jwt from 'jsonwebtoken';
+import { ClientError } from './client-error.js';
+
+export function authMiddleware(req, res, next) {
+  const result = req.get('Authorization');
+  console.log(result);
+  const token = result.split('Bearer ')[1];
```

Comment:
Make sure `result` is defined before calling its `split` method. It will be undefined if there is no Authorization header in the request.